crates/turbopack-node/src/source_map/content_source.rs

```diff
@@ -81,7 +78,15 @@ impl ContentSource for NextSourceMapTraceContentSource {
             self_vc.await?.asset_source,
             NextSourceMapTraceContentProcessorVc::new(id, line, column, frame.name).into(),
         );
-        Ok(wrapped.as_content_source().get(path, Default::default()))
+        Ok(ContentSourceResultVc::exact(
+            ContentSourceContent::Rewrite(
```

Comment:
I don't understand why these need to be rewrites. We're not expecting query params, right? We're not changing headers. This should be equivalent to just doing the `.get()` directly, right?